dist/cjs/uca/UserCollectableAttribute.js

```diff
@@ -166,6 +165,7 @@ const parseAttestableValue = value => {
 function UCABaseConstructor(identifier, value, version) {
   this.timestamp = null;
   this.id = null;
+  this.secureRandom = services.container.SecureRandom;
```

Comment:
Is not better to follow the same pattern as getAnchorService()? Same as before

Reply:
Do we need to attach the secure handler to this? (An UCA instance)

Reply:
Same as the other comment, that pattern that leads to our error... this is an hack
The instance is an singleton, only the wire up is dynamic